<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtWidgets_ProjetClass</class>
 <widget class="QMainWindow" name="QtWidgets_ProjetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>627</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtWidgets_Projet</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="AfficheMessage">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>240</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Afficher Message</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>270</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>IP :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>60</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Port : </string>
    </property>
   </widget>
   <widget class="QPushButton" name="Connexion">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>90</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="IPLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portLineEdit">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="connectionStatusLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status connexion : Déconnecté</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Envoyer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>131</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Envoyer</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>627</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="QtWidgets_Projet.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>AfficheMessage</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgets_ProjetClass</receiver>
   <slot>OnDisplayMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>164</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Connexion</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgets_ProjetClass</receiver>
   <slot>OnConnectButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>115</x>
     <y>134</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Envoyer</sender>
   <signal>clicked()</signal>
   <receiver>QtWidgets_ProjetClass</receiver>
   <slot>OnSendMessageButtonClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>204</y>
    </hint>
    <hint type="destinationlabel">
     <x>313</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>OnDisplayMessageButtonClicked()</slot>
  <slot>OnConnectButtonClicked()</slot>
  <slot>OnSendMessageButtonClicked()</slot>
 </slots>
</ui>
